### PR TITLE
Display Attendee list sorted by Date and Name

### PR DESCRIPTION
**Pull request by sravfeyn**
## Product Description

https://dimagi-dev.atlassian.net/browse/SC-2501

Shows the list of attendees who attended an event sorted by date and name

## Feature Flag

ATTENDANCE_TRACKING

## Safety Assurance

Tests added
### Safety story

Tests added, tested locally

### QA Plan

NA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by Charl1996** [comment]
@sravfeyn Looks good!

I see when there's no event attendees, I get the following:

image

What is your opinion only making the popup possible when the event has started (maybe @kaapstorm you can also chime in here)? The "View attendees" doesn't really have meaning unless the event has either started or is done. 

Also, should we show a message on the popup if there's no attendees captured yet instead of the empty list? There's various places in HQ where we don't show empty lists, but rather some sort of banner. Granted, this is a popup, so I'm not sure how well a banner will show in a popup.

**Comment by sravfeyn** [comment]
Definitely! I am planning to update this once the Attendance taking feature is available.

**Review by Charl1996 — APPROVED** [review]
I don't have any other concerns, so all good from my side.

**Comment by sravfeyn** [comment]
I have updated this to show attendance and edit button only when applicable.

File: corehq/apps/events/models.py [hunk 1]
```diff
@@ -143,36 +179,60 @@ def set_expected_attendees(self, attendee_cases):
...
+                indices=self._get_host_indices(event_host, attendee_host),
                 attrs={
                     'case_type': EVENT_ATTENDEE_CASE_TYPE,
                     'create': True,
                 },
             ))
         self._case_factory.create_or_update_cases(case_structures)
 
-    def _get_ext_cases(self):
+    def mark_attendance(self, attendee_cases, attended_datetime):
```
Comment (sravfeyn) [inline corehq/apps/events/models.py:197]: @kaapstorm Could you please review if this looks okay to you?

**Review by kaapstorm — APPROVED** [review]

**Pull request merged**